src/workflows/airqo_etl_utils/date.py

```
@@ -63,7 +63,7 @@ def get_dag_date_time_values(
                     end_date_time = dag_run.conf["end_date_time"]
                 except Exception as e:
                     exception_occurred = True
-                    print("Exception in get_dag_date_time_values", repr(e))
+                    logger.exception(f"Exception in get_dag_date_time_values", {e})
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)